lib/pure/asynchttpserver.nim

```diff
@@ -70,17 +74,27 @@ type
     maxBody: int ## The maximum content-length that will be read for the body.
     maxFDs: int
 
+proc getPort*(a: AsyncHttpServer): Port {.since: (1, 5, 1).} =
+  ## Returns the port `a` was bound do to.
+  ## Useful when `listen(Port(0))` was called which assigns a port automatically.
+  runnableExamples:
+    let server = newAsyncHttpServer()
+    server.listen(Port(0))
+    doAssert server.getPort.uint16 > 0
+    server.close()
+  result = getLocalAddr(a.socket.getFd, AF_INET)[1]
+
 func getSocket*(a: AsyncHttpServer): AsyncSocket {.since: (1, 5, 1).} =
   ## Returns the `AsyncHttpServer`s internal `AsyncSocket` instance.
   ## 
   ## Useful for identifying what port the AsyncHttpServer is bound to, if it
   ## was chosen automatically.
   runnableExamples:
-    from std/asyncdispatch import Port
```

Comment:
the example could be compiled as standalone before this change, any specific reason why this import was removed?

Reply:
because i added this to this PR:
```
since (1, 5, 1):
  export Port
```
which makes sense; you'd almost always want Port when you use asynchttpserver